eng/tox/tox.ini

```diff
@@ -43,6 +43,7 @@ platform = linux: linux
 passenv = *
 setenv =
   SPHINX_APIDOC_OPTIONS=members,undoc-members,inherited-members
+  PIP_EXTRA_INDEX_URL=https://pypi.python.org/simple
```

Comment:
Is this just working around the issue? Do we still end up with 2 feeds given this one is the extra feed?

Reply:
**yes**.

I'm working on another commit that will actually check the package before installing from the feed. If it's a non-azure-sdk publisher, we will hard error out.

This will happen in `create_package_and_install.py`. The mechanism that we install packages.